tests/integration.rs

```diff
@@ -2,6 +2,23 @@
 
 #![allow(dead_code)]
 
+#[cfg(all(
+    not(miri),
+    not(feature = "gdb"),
```

Comment:
Just out of curiosity, why have we always disabled our tests under gdb?

Reply:
Just tried disabling that, and they all hang with the feature enabled. Probably it's waiting for connection to be established?

Reply:
it's waiting for gdb to connect..